samples/build/build_kaniko-trivy-good_cr.yaml

```
@@ -0,0 +1,17 @@
+---
+apiVersion: shipwright.io/v1alpha1
+kind: Build
+metadata:
+  name: kaniko-trivy-good-build
+  annotations:
+    build.shipwright.io/build-run-deletion: "true"
+spec:
+  source:
+    url: https://github.com/blairdrummond/sample-scanning
+    contextDir: .
```

```suggestion
    url: https://github.com/shipwright-io/sample-scanning
```

Let me know if it was not okay to copy your repo into our org.